lib/stimulus_reflex/utils/sanity_checker.rb

```diff
@@ -49,39 +54,42 @@ def check_caching_enabled
   end
 
   def check_default_url_config
-    unless default_url_config_set?
-      warn_and_exit <<~WARN
-        StimulusReflex strongly suggests that you set default_url_options in your environment files.
-        Otherwise, ActionController and ActionMailer will default to example.com when rendering route helpers.
+    return if StimulusReflex.config.on_missing_default_urls == :ignore
+    if default_url_config_set? == false
+      puts <<~WARN
+        StimulusReflex strongly suggests that you set default_url_options in your environment files. Otherwise, ActionController #{ "and ActionMailer " if defined?(ActionMailer) }will default to example.com when rendering route helpers.
```

Comment:
Is this `#{` supposed to be here?

Reply:
Yeah, I think so! It's probably a little weird looking, but the logic is that we only want to show "and ActionMailer " if ActionMailer is in the project.

Aside from being odd looking, do you see a bigger concern?

Reply:
Yea thats what I figured, just wanted to double check just because it caught my eye.